Sources/Interaction/UI/VolumeController/index.js

```diff
@@ -251,6 +259,15 @@ function vtkVolumeController(publicAPI, model) {
     }
   };
 
+  const rescaleColorMap = publicAPI.setRescaleColorMap;
+  publicAPI.setSescaleColorMap = (value) => {
```

Comment:
`setSescaleColorMap`?  What is this function doing?

Reply:
oups, should be `setRescaleColorMap`

Reply:
true/false so either the full data range is used for the color map or only the opacity range.

Reply:
Ah, thanks for the explanation :+1:

Reply:
Do you mind fixing it? thx!

Reply:
Yes, will do :-)